content/backmatter.tex

```diff
@@ -444,60 +434,61 @@ \subsection{SHMEM\_CACHE}
-    \_SHMEM\_REDUCE\_SYNC\_SIZE & \_SHMEM\_VENDOR\_STRING \\
+    \CONST{\_SHMEM\_SYNC\_VALUE}         & \CONST{\_SHMEM\_REDUCE\_MIN\_WRKDATA\_SIZE} \\
+    \CONST{\_SHMEM\_BARRIER\_SYNC\_SIZE} & \CONST{\_SHMEM\_MAJOR\_VERSION} \\
+    \CONST{\_SHMEM\_BCAST\_SYNC\_SIZE}   & \CONST{\_SHMEM\_MINOR\_VERSION} \\
+    \CONST{\_SHMEM\_COLLECT\_SYNC\_SIZE} & \CONST{\_SHMEM\_MAX\_NAME\_LEN} \\
+    \CONST{\_SHMEM\_REDUCE\_SYNC\_SIZE}  & \CONST{\_SHMEM\_VENDOR\_STRING} \\
 \end{tabular}
 \end{center}
 do not adhere to the \Cstd standard's reserved identifiers and the \Cpp
 standard's reserved names.  These constants have been deprecated and replaced
 with corresponding constants of prefix \shmemprefix{} that adhere to \CorCpp{}
 and \Fortran naming conventions.
 
-\subsection{\FUNC{shmem\_fetch}, \FUNC{shmem\_set}, \FUNC{shmem\_cswap},
+\subsection{\VAR{SMA\_*} Environment Variables}\label{subsec:deprecate-sma-env}
+The environment variables \VAR{SMA\_VERSION}, \VAR{SMA\_INFO},
+\VAR{SMA\_SYMMETRIC\_SIZE}, and \VAR{SMA\_DEBUG}
+were deprecated in order to normalize the \openshmem \ac{API} to use
+\shmemprefix{} as the standard prefix for all environment variables.
+
+\subsection{\CorCpp: \FUNC{shmem\_wait}}
+The \CorCpp interface for \FUNC{shmem\_wait} was identified as unintuitive with respect to
+the comparison operation it performed.  As \FUNC{shmem\_wait} can be trivially
+replaced by \FUNC{shmem\_wait\_until} where \VAR{cmp} is
+\CONST{SHMEM\_CMP\_NE}, the \FUNC{shmem\_wait} interface was deprecated in
+favor of \FUNC{shmem\_wait\_until}, which makes the comparison operation
+explicit and better communicates the developer's intent.
+
+\subsection{\CorCpp: \FUNC{shmem\_fetch}, \FUNC{shmem\_set}, \FUNC{shmem\_cswap},
   \FUNC{shmem\_swap}, \FUNC{shmem\_finc}, \FUNC{shmem\_inc},
   \FUNC{shmem\_fadd}, \FUNC{shmem\_add}}
 
 The \CorCpp interfaces for \FUNC{shmem\_fetch}, \FUNC{shmem\_set},
 \FUNC{shmem\_cswap}, \FUNC{shmem\_swap}, \FUNC{shmem\_finc}, \FUNC{shmem\_inc},
 \FUNC{shmem\_fadd}, and \FUNC{shmem\_add} were deprecated and replaced with
-similarly-named interfaces within the \FUNC{shmem\_atomic\_*} namespace
+similarly named interfaces within the \FUNC{shmem\_atomic\_*} namespace
 in order to more clearly identify these calls as performing atomic operations.
 In addition, the abbreviated names ``cswap'', ``finc'', and ``fadd'' were
 expanded for clarity to ``compare\_swap'', ``fetch\_inc'', and ``fetch\_add''.
 
-\subsection{SMA\_* Environment Variables}
-The environment variables
-\begin{center}
-\begin{tabular}{ll}
-    SMA\_VERSION \\
-    SMA\_INFO \\
-    SMA\_SYMMETRIC\_SIZE \\
-    SMA\_DEBUG \\
-\end{tabular}
-\end{center}
-were deprecated in order to normalize the \openshmem \ac{API} to use
-\shmemprefix{} as the standard prefix for all environment variables.
-
-\subsection{Fortran API}\label{subsec:deprecate-fortran}
+\subsection{Fortran API}\label{subsec:deprecate-fortran} %% WARNING: Issue #66.
```

Comment:
`Fortran` -> `\Fortran`? Although, this may break the PDF TOC entry here, which means it should be `\textit{Fortran}`.